jaxley/modules/base.py

```diff
@@ -2490,7 +2495,7 @@ def _set_inds_in_view(
             ].unique()
             pre = base_edges["pre_global_comp_index"].isin(incl_comps).to_numpy()
             post = base_edges["post_global_comp_index"].isin(incl_comps).to_numpy()
-            possible_edges_in_view = base_edges.index.to_numpy()[(pre & post).flatten()]
+            possible_edges_in_view = base_edges.index.to_numpy()[(pre | post).flatten()]
```

Comment:
@michaeldeistler @jnsbck this is the culprit for #533 ! Essentially if you're indexing `net.cell(0)`, then it's impossible for a view of that cell to include the compartments of another cell (unless it's an autapse)

I think this is a question of how you want this to behave. I've changed it to an "or" condition, so `net.cell(0).edges` will include both synapses onto this cell, as well as synapses it makes with other cells

For the purposes of this PR, it would make sense to only include the edges of the post synaptic cell, however you could argue for different expected behaviors.

Reply:
I decided to just go forward with only appending the post synaptic target so that I could finish the tasks for this PR. We can discuss if that's actually the desired behavior for the `View`